modular_skyrat/modules/shotgunrebalance/code/shotgun.dm

```diff
@@ -0,0 +1,274 @@
+/* /obj/projectile/bullet/pellet
+	var/tile_dropoff = 0.45
+	var/tile_dropoff_s = 0.25 */
+
+/obj/item/ammo_casing/shotgun
+	icon = 'modular_skyrat/modules/shotgunrebalance/icons/shotshells.dmi'
+	custom_materials = list(/datum/material/iron=1000) //We will be using this to prevent refund scamming mats
```

Comment:
```suggestion
	custom_materials = list(/datum/material/iron = 1000) //We will be using this to prevent refund scamming mats
```
Do the same for the others.